Lesson6/Task2/Program.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2 {
+    abstract class Figure {
+        public enum Color {
+            Black,
+            Red,
+            Green,
+            Yellow,
+            Blue,
+            Magenta,
+            Cyan,
+            White
+        }
+
+        protected Color color;
+        
+        protected bool visible;
+
+        public abstract void MoveHorisontal(double distance);
+
+        public abstract void MoveVertical(double distance);
+
+        public virtual double Area() => 0;
```

Comment:
```suggestion
        public virtual double GetArea() => 0;
```
Иначе делайте свойство.

Reply:
да, можно переименовать* спасибо